core/templates/dev/head/pages/exploration-editor-page/improvements-tab/improvements-tab.directive.ts

```diff
@@ -58,11 +58,13 @@ angular.module('oppia').directive('improvementsTab', [
 
           $scope.onlyShowOpenTasks = true;
 
-          $scope.getStatusCssClass =
-            ImprovementsDisplayService.getStatusCssClass;
+          $scope.getStatusCssClass = function(status) {
```

Comment:
Type?

Reply:
This isn't migrated yet. I will add the types when I migrate this.